aeneas/src/ir/SsaNormalizer.v3

```diff
@@ -692,6 +692,7 @@ class SsaRaNormalizer extends SsaRebuilder {
 		var tn = normTypeArg(op, 0);
 		var refs = genRefs(oldApp.inputs);
 		if (!VariantNorm.?(tn) && V3.isVariant(tn.newType)) return map1(oldApp, normVariantEqual(oldApp, tn.newType, refs[0], refs[1]));
```

Comment:
Might be worth doing a match on the `tn` type here since there are two cases now.